apps/web/src/modules/proposal/components/ProposalActions/CancelAction.tsx

```diff
@@ -30,26 +30,7 @@ const Cancel: React.FC<{
   )
 }
 
-const Veto: React.FC<{
-  proposalId: string
-  onSuccess: () => void
-}> = (props) => {
-  return (
-    <GovernorContractButton
-      functionName="veto"
-      args={[props.proposalId as Address]}
-      buttonText="Veto"
-      buttonClassName={proposalActionButtonVariants['veto']}
-      {...props}
-    />
-  )
-}
-
-export const OwnerActions: React.FC<OwnerActionsProps> = ({
-  proposalId,
-  showCancel,
-  showVeto,
-}) => {
+export const CancelAction: React.FC<CancelActionProps> = ({ proposalId }) => {
```

Comment:
super nit: maybe would rename this `CancelButton` to distinguish a bit between the infobox-with-button components (e.g. `SuccessfulProposalActions` and `VetoAction`) and this one which is just a button and success modal.